ntbs-service/appsettings.json

```diff
@@ -20,5 +20,17 @@
   },
   "MigrationConfig": {
     "DateRangeJobIntervalInMonths": 2
+  },
+  "ScheduledJobConfig": {
+    "UserSyncEnabled": true,
+    "UserSyncTiming": "0 3 * * *",
```

Comment:
I think I would just call them what they are - cron strings, so `UserSyncCron` - it's a widely enough understood term I reckon?